shdr_builder.cc

```diff
@@ -85,13 +85,14 @@ void ShdrBuilder::Freeze() {
     }
 }
 
-void ShdrBuilder::RegisterShdr(Elf_Off offset, uint64_t size, ShdrType type, uint64_t entsize) {
+void ShdrBuilder::RegisterShdr(Elf_Off offset, uint64_t size, ShdrType type, uint64_t entsize, Elf_Word info) {
     Elf_Shdr shdr = {0};
     shdr.sh_name = GetShName(type);
     shdr.sh_offset = offset;
     shdr.sh_addr = offset;
     shdr.sh_size = size;
     shdr.sh_entsize = entsize;
+    shdr.sh_info = info;
```

Comment:
Let's fill other missing fields as well. `sh_info` and `sh_addralign`?

Reply:
Thank you. I will do it in another PR.